test/db/cmd/cmd_ag

```diff
@@ -0,0 +1,337 @@
+NAME=agi
+FILE=bins/elf/hello_world
+CMDS=<<EOF
+aa 2> /dev/null
+agi*
+EOF
+EXPECT=<<EOF
+agn "sym.imp.free" base64:
+agn "0x0000083f" base64:
+agn "_ITM_deregisterTMCloneTable" base64:
+agn "sym.imp.strcpy" base64:
+agn "0x00000814" base64:
+agn "sym.imp.puts" base64:
+agn "0x00000833" base64:
+agn "sym.imp.strlen" base64:
+agn "0x000007cf" base64:
+agn "0x000007de" base64:
+agn "__libc_start_main" base64:
+agn "__gmon_start__" base64:
+agn "sym.imp.malloc" base64:
+agn "0x000007f6" base64:
+agn "sym.imp.strcat" base64:
+agn "0x00000827" base64:
+agn "_ITM_registerTMCloneTable" base64:
+agn "section..plt.got" base64:
+agn "0x0000077e" base64:
+age "0x0000083f" "sym.imp.free"
+age "0x00000814" "sym.imp.strcpy"
+age "0x00000833" "sym.imp.puts"
+age "0x000007cf" "sym.imp.strlen"
+age "0x000007de" "sym.imp.strlen"
+age "0x000007f6" "sym.imp.malloc"
+age "0x00000827" "sym.imp.strcat"
+age "0x0000077e" "section..plt.got"
+EOF
+RUN
+
+
+NAME=agx
+FILE=bins/elf/hello_world
+CMDS=<<EOF
+aa 2> /dev/null
+agx* @ main
+EOF
+EXPECT=<<EOF
+agn "main" base64:
+agn "0x000006bd" base64:
+age "0x000006bd" "main"
+EOF
+RUN
+
+NAME=formats
+FILE=bins/elf/hello_world
+CMDS=<<EOF
+aa 2> /dev/null
+agx @ main
+agid
+agig
+agij
+agik
+agit
+EOF
+EXPECT=<<EOF
+.--------------------.
+|  0x000006bd        |
+`--------------------'
+    v
+    |
+    |
+.--------------------.
+|  main              |
+`--------------------'
+digraph code {
+rankdir=LR;
+outputorder=edgesfirst
+graph [bgcolor=azure];
+edge [arrowhead=normal, color="#3030c0" style=bold weight=2];
+node [fillcolor=white, style=filled shape=box fontname="Courier" fontsize="8"];
+0 [URL="sym.imp.free", color="lightgray", label="sym.imp.free"]
```

Comment:
master here uses "sym.imp.free" instead of 0. Is it ok?

Reply:
Why not? Its valid dot file. Less copies of the the same string, less place in code that needs to properly print those strings. Allow this function to be used with graphs that don't have unique titles.